e2e/dr_test.go

```diff
@@ -53,7 +54,7 @@ func TestDR(dt *testing.T) {
 			panic(err)
 		}
 
-		deployer, err := deployers.New(tc.Deployer)
+		deployer, err := deployers.New(deploySpecs[tc.Deployer])
```

Comment:
I will better to get the deployer from the map and panic like we do for pvcspec above. It should never happen since we validate the config, but if validation is broken this can return the zero value of deployer config which will fail in a strange and hard to debug way.